package.json

```diff
@@ -8,8 +8,8 @@
     "test:okta-hosted-login": "protractor okta-oidc-tck/e2e-tests/okta-hosted-login/conf.js",
     "custom-login-server": "npm start --prefix custom-login/",
     "test:custom-login": "protractor okta-oidc-tck/e2e-tests/custom-login/conf.js",
-    "resource-server": "mvn -Dokta.oauth2.issuer=https://${OKTA_DOMAIN}.com/oauth2/default -f samples-java-spring-mvc/resource-server/pom.xml",
-    "test:e2e": "cross-env TEST_TYPE=implicit && npm run get-oidc-tck && npm run test:okta-hosted-login && npm run test:custom-login",
```

Comment:
`cross-env` doesn't export env vars in the context of the new process started by TCK. `export` works fine but needs to be changed for support on windows later.